packages/node_modules/node-red/package.json

```diff
@@ -40,15 +40,13 @@
         "express": "4.17.1",
         "fs-extra": "9.1.0",
         "node-red-admin": "^0.2.6",
-        "node-red-node-rbe": "^0.5.0",
-        "node-red-node-tail": "^0.3.0",
         "nopt": "5.0.0",
         "semver": "7.3.5"
     },
     "optionalDependencies": {
-        "bcrypt": "3.0.6"
+        "bcrypt": "5.0.1"
```

Comment:
Not ideal to have this change mixed in here as it's nothing to do with anything here... same for the engines.node updates. Yes they need updating, but in their own commit.